packages/faustwp-core/src/wpHooks/overloads.ts

```diff
@@ -68,6 +69,16 @@ type FaustCoreFilters = {
     callback: (wpUrl: string, context: Record<string, never>) => string,
     priority?: number | undefined,
   ): void;
+
+  addFilter(
+    hookName: 'toolbarNodes',
+    namespace: string,
+    callback: (
+      toolbarNodes: FaustToolbarNodes,
+      context: { seedNode: SeedNode },
```

Comment:
Since `seedNode` could be undefined:

```suggestion
      context: { seedNode?: SeedNode },
```